setup.py

```diff
@@ -29,14 +29,15 @@
         saq=saq.__main__:main
     """,
     install_requires=[
-        "redis>=4.2,<5.0",
+        "redis>=4.2,<6.0",
         "croniter>=0.3.18",
     ],
     extras_require={
         "hiredis": ["redis[hiredis]>=4.2.0"],
         "web": ["aiohttp", "aiohttp_basicauth"],
         "dev": [
             "black",
+            "green",
```

Comment:
i'd rather not introduce green and stick to unittest

Reply:
Cool, done.